tests/profiles/adobefonts_test.py

```diff
@@ -70,22 +88,33 @@ def test_check_find_empty_letters():
     """Validate that empty glyphs are found."""
     check = CheckTester(adobefonts_profile, "com.adobe.fonts/check/find_empty_letters")
 
-    # this font has inked glyphs for all letters
-    font = TEST_FILE("source-sans-pro/OTF/SourceSansPro-Regular.otf")
-    assert_PASS(check(font))
+    PASS_MSG = "No empty glyphs for letters found."
+
+    # this OT-CFF font has inked glyphs for all letters
+    ttFont = TTFont(TEST_FILE("source-sans-pro/OTF/SourceSansPro-Regular.otf"))
+    assert assert_PASS(check(ttFont)) == PASS_MSG
+
+    # this OT-CFF2 font has inked glyphs for all letters
+    ttFont = TTFont(TEST_FILE("source-sans-pro/VAR/SourceSansVariable-Italic.otf"))
+    assert assert_PASS(check(ttFont)) == PASS_MSG
+
+    # this TrueType font has inked glyphs for all letters
+    ttFont = TTFont(TEST_FILE("source-sans-pro/TTF/SourceSansPro-Bold.ttf"))
+    assert assert_PASS(check(ttFont)) == PASS_MSG
 
     # this font has empty glyphs for several letters,
     # the first of which is 'B' (U+0042)
-    font = TEST_FILE("familysans/FamilySans-Regular.ttf")
-    message = assert_results_contain(check(font), FAIL, "empty-letter")
-    assert message == "U+0042 should be visible, but its glyph ('B') is empty."
+    ttFont = TTFont(TEST_FILE("familysans/FamilySans-Regular.ttf"))
+    msg = assert_results_contain(check(ttFont), FAIL, "empty-letter")
+    assert msg == "U+0042 should be visible, but its glyph ('B') is empty."
```

Comment:
In general there's no need to instantiate a TTFont if you're not going to manipulate it in the code-test. A simple font file path was good enough already.

But it is also not a big deal, so feel free to do it as you like.

Reply:
Thanks for the insight. Will leave it as-is.